src/frontend/sharedComponents/EditScreen/ActionButtons.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {useCallback} from 'react';
+import {defineMessages, useIntl} from 'react-intl';
+import {ActionTab} from '../ActionTab';
+import PhotoIcon from '../../images/observationEdit/Photo.svg';
+import AudioIcon from '../../images/observationEdit/Audio.svg';
+import DetailsIcon from '../../images/observationEdit/Details.svg';
+import {BottomSheetModal, useBottomSheetModal} from '../BottomSheetModal';
+import {PermissionAudio} from '../PermissionAudio';
+import {Audio} from 'expo-av';
+import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes';
+
+const m = defineMessages({
+  audioButton: {
+    id: 'screens.ObservationEdit.ObservationEditView.audioButton',
+    defaultMessage: 'Audio',
+    description: 'Button label for adding audio',
+  },
+  photoButton: {
+    id: 'screens.ObservationEdit.ObservationEditView.photoButton',
+    defaultMessage: 'Photo',
+    description: 'Button label for adding photo',
+  },
+  detailsButton: {
+    id: 'screens.ObservationEdit.ObservationEditView.detailsButton',
+    defaultMessage: 'Details',
+    description: 'Button label to add details',
+  },
+});
+
+interface ActionButtonsProps {
+  showAudio: boolean;
+  fieldIds?: string[];
+}
+
+export const ActionButtons = ({showAudio, fieldIds}: ActionButtonsProps) => {
```

Comment:
due to the issues I'm encountering when testing this, might make sense to remove all of the audio-related code here (and in `EditScreen/index.tsx`) for now, and let that get re-introduced with the audio work I'm doing. reason is that I don't think the issues are related to this PR, but could be mistaken.

thoughts?